### PR TITLE
Various Python layer enhancements

### PR DESCRIPTION
**Pull request by jschueller**
- Use one entry for all contour iso-values using legend attribute not labels, like the R backend
- Do not set Pairs text color to point color (keep it black by default)
- Add operator*(Scalar, Sample)

### CODE REVIEW

**Comment by regislebrun** [comment]
+1

File: python/test/t_Sample_operators.expout [hunk 1]
```diff
@@ -12,6 +12,10 @@ sample*2:     [ a0 a1 ]
 0 : [ 60 80 ]
 1 : [ 62 82 ]
 2 : [ 64 84 ]
+2*sample:     [ a0 a1 ]
+0 : [ 60 80 ]
+1 : [ 62 82 ]
+2 : [ 64 84 ]
 sample/2:     [ a0   a1   ]
```
Comment (sofianehaddad) [inline python/test/t_Sample_operators.expout]: @jschueller  f9796be should be in a new PR ?
Btw seems a bug here no ? 2*Sample == Sample ???

Comment (jschueller) [inline python/test/t_Sample_operators.expout]: no:
```sample5 =     [ a0 a1 ]
 0 : [ 30 40 ]
 1 : [ 31 41 ]
2 : [ 32 42 ]
```

Comment (sofianehaddad) [inline python/test/t_Sample_operators.expout]: ok but I think the commit should be outside this PR

Comment (jschueller) [inline python/test/t_Sample_operators.expout]: I renamed the PR: Various Python layer enhancements

**Comment by jschueller** [comment]
it think it's reasonable : we have already scalar*point, scalar*matrix, also np.array supports this

**Comment by jschueller** [comment]
So, what do you think @dbarbier ?

**Comment by dbarbier** [comment]
We define Sample+int, but neither Point+int nor int+Sample are defined.  IMO we should be consistent.  But that is not a big deal for me, merge it if you want.

**Comment by jschueller** [comment]
I agree sample+int is weird, I'll try to address that in another PR

**Pull request merged**